backend/test/integration/openai.test.ts

```diff
@@ -176,22 +175,32 @@ describe('OpenAI Integration Tests', () => {
 		expect(chatResponse.completion?.content).toBe('Hi');
 		// check the chat history has been updated
 		expect(chatHistory.length).toBe(3);
-		// system role is added to the start of the chat history
-		expect(chatHistory[0].completion?.role).toBe('system');
-		expect(chatHistory[0].completion?.content).toBe(systemRoleDefault);
-		// rest of the chat history is in order
-		expect(chatHistory[1].completion?.role).toBe('user');
-		expect(chatHistory[1].completion?.content).toBe("I'm a user");
-		expect(chatHistory[2].completion?.role).toBe('assistant');
-		expect(chatHistory[2].completion?.content).toBe("I'm an assistant");
+		expect('completion' in chatHistory[0]);
+		expect('completion' in chatHistory[1]);
+		expect('completion' in chatHistory[2]);
+		if (
+			'completion' in chatHistory[0] &&
+			'completion' in chatHistory[1] &&
+			'completion' in chatHistory[2]
+		) {
```

Comment:
As per previous comment, this seems a bit awkward. Also, I'd be tempted to test each message separately, in its own if...else block.

Reply:
no longer relevant, we now just do one big expect statement for the whole of the history